docs/domains/index.html

```diff
@@ -1029,6 +1029,10 @@
           "description": "The name of this cluster. Required",
           "type": "string"
         },
+        "allowReplicasBelowMinDynClusterSize": {
```

Comment:
If true (the default), the number -> If true (the default), then the number
that the number of replicas is not below the minimum dynamic cluster -> that the number of replicas is not less than the minimum dynamic cluster setting.

Reply:
ditto